framework/src/main/java/org/checkerframework/framework/flow/CFAbstractAnalysis.java

```diff
@@ -169,4 +189,24 @@ public V createSingleAnnotationValue(AnnotationMirror anno, TypeMirror underlyin
         annos.add(anno);
         return createAbstractValue(annos, underlyingType);
     }
+
+    /**
+     * Get the types utility.
+     *
+     * @return {@link #types}
+     */
+    @SuppressWarnings("deprecation")
```

Comment:
Is there a deprecation warning because this method overrides the deprecated supermethod?
Maybe also add a TODO to the supermethod so that we don't forget to remove this suppression once we remove that method.

Reply:
Yes, the deprecation warning is caused by overriding the deprecated method.